### ISSUE TITLE
Collections page is not working.

### ISSUE THREAD

**Issue by GermanUrrego** [bitwarden/web#1473]
### Steps To Reproduce

1. Assign "Manager" role to a user and select  'This user can access and modify all items. '
2. Click on 'Organization' and later in 'Manage'.

### Expected Result

Collections should be accessible in the Manage Tab. Otherwise 'Admin' rights are needed to be shown properly.

### Actual Result

* The spinning wheel is shown endlessly
 
![image](https://user-images.githubusercontent.com/99652117/153849271-91d11999-e401-4ba9-8184-755fe9e46a12.png)
*  Others options in the 'Manage' Tab are available i.e People, Groups.
*  Also in the Console of the browser, this error is shown:
`ERROR Error: Uncaught (in promise): t: {"response":{"Object":"error","Message":"Resource not found.","ValidationErrors":null,"ExceptionMessage":null,"ExceptionStackTrace":null,"InnerExceptionMessage":null},"message":"Resource not found.","validationErrors":null,"captchaRequired":false,"statusCode":404}`

### Screenshots or Videos

![image](https://user-images.githubusercontent.com/99652117/153849271-91d11999-e401-4ba9-8184-755fe9e46a12.png)

### Operating System

Linux

### Operating System Version

Debian 4.19.208-1

### Web Browser

Chrome

### Browser Version

98.0.4758.82

### Build Version

2.25

**Comment by Hinton** [comment]
We should handle this more gracefully. I suspect the issue occurs due to the user permission being updated which doesn't reflect until the user has logged out and back in again or performed a full sync.

**Comment by Hinton** [comment]
@GermanUrrego does the error occur after logging out and back in again?

**Comment by GermanUrrego** [comment]
@Hinton yes. The error occur after logging out and log back in. Even trying `Custom` permission, when selecting `Manage All Collections`, the changes on the Collections are not preserved after saving them. the other fields such as `Groups` and `User` can be managed.

**Comment by Hinton** [comment]
Can you verify the issue exists in the latest version of the server and web vault?

**Comment by GermanUrrego** [comment]
I've seen a similar error in the server version issues (#1117) . I cannot verify at the moment with the latest version but the Server version is 1.45.4 and Web Vault version is 2.25.0.

**Comment by Hinton** [comment]
@GermanUrrego That's a different error caused by some logic within the page. The error message from your console claims that the server returns a "Resource not found" error which should only happen when the user does not have access to view collections.

Just to confirm this is how the user permission looks:
![image](https://user-images.githubusercontent.com/137855/156171492-bfb3eafe-a99c-4489-8337-334b4e126f93.png)

**Comment by GermanUrrego** [comment]
Hi @Hinton, thank you for the clarification. What I meant was the similarity in the loading process and the fact that even the custom setting are not preserved, in my case, the collection option over `Custom` are not preserved.

**Comment by Hinton** [comment]
Since the issue seems to be resolved now, I'm going to close this issue.
